data/strings/strings.txt

```diff
@@ -16619,7 +16621,7 @@
 
   [big_font]
     tags = android,ios
-    en = Increase font size on the map
+    en = Increase map label size
```

Comment:
No. It is less clear.

Reply:
see https://github.com/organicmaps/organicmaps/pull/5059/commits/ca0bf3d1de80b67b4d9d37f5a78418d9a6cff58c#r1183019239